docs/authorization.md

```diff
@@ -130,28 +129,34 @@ if (! $user->hasPermission('users.create')) {
 }
 ```
 
-#### Authorizing via Filters
+#### Authorizing via Routes
 
-You can restrict access to multiple routes through a [Controller Filter](https://codeigniter.com/user_guide/incoming/filters.html). One is provided for both restricting via groups the user belongs to, as well as which permission they need. The filters are automatically registered with the system under the `group` and `permission` aliases, respectively. You can define the protections within **app/Config/Filters.php**:
+You can restrict access to a route or route group through a
+[Controller Filter](https://codeigniter.com/user_guide/incoming/filters.html).
 
-```php
-public $filters = [
-    'group:admin,superadmin' => ['before' => ['admin/*']],
-    'permission:users.manage' => ['before' => ['admin/users/*']],
-];
-```
+One is provided for restricting via groups the user belongs to, the other
+is for permission they need. The filters are automatically registered with the
+system under the `group` and `permission` aliases, respectively.
 
-#### Authorizing via Routes
-
-The filters can also be used on a route or route group level:
+You can set the filters within **app/Config/Routes.php**:
 
 ```php
 $routes->group('admin', ['filter' => 'group:admin,superadmin'], static function ($routes) {
-    $routes->resource('users');
+    $routes->group(
+        '',
+        ['filter' => ['group:admin,superadmin', 'permission:users.manage']],
```

Comment:
```suggestion
        ['filter' => ['group:admin,superadmin', 'permission:users.manage-admins']],
```

Reply:
`permission:users.manage` is not listed in https://github.com/codeigniter4/shield/blob/develop/docs/authorization.md#defining-available-permissions
But in this case,  I feel `permission:users.manage-admins` is a bit weird.
The URI seems the page to manage users, not admin-users.